src/Button/DigitizeButton/DigitizeButton.tsx

```diff
@@ -436,7 +452,12 @@ class DigitizeButton extends React.Component<DigitizeButtonProps, DigitizeButton
       map
     } = this.props;
 
-    this._interactions.forEach(i => map.removeInteraction(i));
+    map.removeInteraction(this._drawInteraction);
+    map.removeInteraction(this._selectInteraction);
+    map.removeInteraction(this._modifyInteraction);
+    map.removeInteraction(this._translateInteraction);
```

Comment:
Should we also `null`ify the references? è.g.

```
    map.removeInteraction(this._drawInteraction);
    map.removeInteraction(this._selectInteraction);
    map.removeInteraction(this._modifyInteraction);
    map.removeInteraction(this._translateInteraction);

    // nullify
   this._drawInteraction = null; //…
```

Reply:
Since the component is unmounted at the time, it's actually not necessary.